cmd/umserver/main.go

```diff
@@ -27,8 +30,22 @@ func main() {
 		closers     []io.Closer
 	)
 
+	cfg, err := config.NewConfig()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Example
+	fmt.Println(cfg)
+	fmt.Println(cfg.GraylogConfig(ctx))
+	fmt.Println(cfg.PostgresConfig(ctx))
```

Comment:
There is already imported log package, please use it.

Reply:
Thanks, I fixed it